.github/workflows/create-releases.yml

```diff
@@ -0,0 +1,42 @@
+name: Create releases
+on:
+  schedule:
+    - cron: '0 5 * * *' # every day at 5am UTC
```

Comment:
not sure if we need to publish at 5 AM every day haha, but everything else looks sound!